packages/bpk-component-dialog/src/bpk-dialog.scss

```diff
@@ -0,0 +1,30 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+.bpk-dialog__container {
```

Comment:
What's this style for? It shouldn't be required.

Reply:
Without this style the dialog would go to the top of the screen.

Reply:
The styles should be encapsulated in the modal component and exposed as a prop in that case. If we override modal styles from outside the modal component then when we make updates to the modal styles in the future its hard to know if we break the dialog styles.